app/lib/setUpRendererAndDrawChords.ts

```diff
@@ -44,7 +44,9 @@ export const setupRendererAndDrawChords = (
   }
```

Comment:
Line 43 crashes the page with the following error: "TypeError: setStaves is not a function"